GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -115,20 +117,16 @@ - (NSString *)clientID {
 }
 
 - (nullable NSString *)hostedDomain {
-  NSString *idToken = [self idToken];
-  if (idToken) {
-    OIDIDToken *idTokenDecoded = [[OIDIDToken alloc] initWithIDTokenString:idToken];
+  NSString *lastTokenResponseToken = [self lastTokenResponseToken];
```

Comment:
Would probably be clearer to use `self.idToken.tokenString` here.

Reply:
Good idea. It is clearer.